matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/secrets/DefaultSharedSecretStorageService.kt

```diff
@@ -385,7 +385,12 @@ internal class DefaultSharedSecretStorageService @Inject constructor(
         return IntegrityResult.Success(keyInfo.content.passphrase != null)
     }
 
+    @Deprecated("Requesting custom secrets not yet support by rust stack, prefer requestMissingSecrets")
```

Comment:
I do not think it is useful to have the `@Deprecated` here. It's in the interface.

Reply:
ktlint complains if it's not there also